packages/react-native-editor/bin/generate-pot-files.sh

```diff
@@ -204,10 +204,12 @@ fi
 # Set JS bundle directory
 BUNDLE_DIR="$TARGET_PATH/bundle"
 mkdir -p $BUNDLE_DIR
+trap '{ rm -rf -- "$BUNDLE_DIR"; }' EXIT
 
 # Set source files extraction directory
 EXTRACT_SOURCE_FILES_DIR="$TARGET_PATH/source-files"
 mkdir -p $EXTRACT_SOURCE_FILES_DIR
+trap '{ rm -rf -- "$EXTRACT_SOURCE_FILES_DIR"; }' EXIT
```

Comment:
The bundle and extract source files directories are only meant to hold intermediate files so they should be removed when the script finishes or fails.